### ISSUE TITLE
[Upcoming] Adding new targeted landing pages to site

### ISSUE THREAD

**Issue by tealtigrrr** [code-423n4/code423n4.com#1033]
Placeholder issue for some new content we'll be adding to the site:
From @sockdrawermoney:

> We're going to want to add some content-focused pages to be added to the C4 site. these will be generally similar pages in structure/design and could easily just take off from the design we have for other pages, but will each one be specific to lead sources who want to have a way to make a friendly invite to their portfolio companies. 
> 
> @🧺 realfakepicnic is working on this content-wise but we'll need the pages created.

**Comment by tealtigrrr** [comment]
@sockdrawermoney Just confirming, we're waiting on content for these before we can move forward? Thanks!

**Comment by tealtigrrr** [comment]
@sockdrawermoney Is this something we're still considering adding to the site?

**Comment by SamusElderg** [comment]
Closing this as no longer relevant
